prime-router/docs/design/design/yaml-validation/yaml-intellij-setup.md

```diff
@@ -0,0 +1,22 @@
+# Intellij Setup
+
+Getting Intellij setup to report errors during coding greatly speeds up development and reduces errors.
+
+## Apply Schema
+
+Add the following comment to the top of your yaml file to enable Intellij validation. Ensure the path to the schema 
+is correct.
+```yaml
+# $schema: ./../../../metadata/json_schema/schema.json
+```
+You can also register the schema with Intellij and apply it to certains files and directories. I don't recommend doing
+it this way as tracking it in the files ensure others don't have to continually update their Intellij settings to stay
+up to date on schema locations and new files.
+
+## Set up red underline on errors
+
+- Preferences -> Editor -> Inspections
```

Comment:
Maybe add these steps to the developer getting started docs as well?

Reply:
can do